pages/linux/extundelete.md

```diff
@@ -0,0 +1,21 @@
+# extundelete
+
+> Restore files from an unmounted ext3 or ext4 partition.
```

Comment:
Maybe this is more clear:
```suggestion
> Recover deleted files from ext2, ext3 or ext4 partitions by parsing the journal.
```

Reply:
I will add the rest of the suggestion, but this utility does not work with ext2 filesystems as they lack a journal.
```suggestion
> Recover deleted files from ext3 or ext4 partitions by parsing the journal.
```

Reply:
😮 so, Arch Linux has a bug in their package description https://www.archlinux.org/packages/community/x86_64/extundelete/